.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Kindly add `test.md` in `.gitignore` so that it will be ignored by git and not be pushed into your repo. Kindly delete it from your  repo as it is meant to be ignored by git.